web/src/app/review/page.tsx

```diff
@@ -11,42 +15,132 @@ import {
 
 import { Button } from '@/components/ui/button';
 
+import { getCardsCardsGet, createReviewReviewsPost } from '@/gen';
+import { useState } from 'react';
+
 export default function Review() {
+    const {
+        isPending,
+        isError,
+        data: dueCards,
+    } = useQuery({
+        queryKey: ['cards', 'due'],
+        queryFn: () => getCardsCardsGet({ query: { only_due: true } }),
+    });
+
+    const queryClient = useQueryClient();
+
+    const reviewCard = useMutation({
+        mutationFn: (feedback: 'ok' | 'skipped' | 'forgot') =>
+            createReviewReviewsPost({
+                body: {
+                    card_id: activeCard!.id,
+                    feedback: feedback,
+                },
+            }),
+        onSuccess: () => {
+            queryClient.invalidateQueries({ queryKey: ['cards'] });
+        },
+        // TODO: implement error handling
+    });
+
+    const [activeCardIndex, setActiveCardIndex] = useState(0);
+
+    const activeCard = dueCards?.data?.[activeCardIndex];
+    const isActiveCardOverdue =
```

Comment:
this is fine for now but we could/should move this to the backend and include a flag on cards that are >1 day overdue